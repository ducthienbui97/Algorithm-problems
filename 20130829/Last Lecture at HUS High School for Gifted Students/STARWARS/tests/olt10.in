1000
5002 313 5167 203 S
1695 4728 1772 4547 S
7534 1411 7622 1236 N
991 7527 1063 7373 E
5610 1269 5760 1138 S
995 4335 1148 4225 S
3548 5770 3727 5608 W
4098 1077 4176 927 S
6971 1067 7094 912 E
4437 4104 4586 3936 E
5123 7622 5260 7512 E
1575 3249 1711 3123 N
4729 1355 4833 1273 S
3551 2297 3631 2159 S
4596 748 4779 555 S
864 4146 988 3979 N
2893 2411 2963 2299 S
7280 376 7384 286 N
1366 1766 1544 1647 E
5138 4806 5205 4735 E
4700 3708 4873 3623 W
3336 6718 3433 6600 E
6600 5550 6754 5407 S
2368 2103 2420 2044 W
4391 2022 4480 1863 E
634 4597 825 4545 N
7695 6819 7807 6705 E
2911 1181 3010 1122 W
2871 5454 3014 5379 W
237 7255 404 7173 N
5097 6091 5282 6004 E
6577 2190 6702 2112 S
6324 1430 6404 1362 N
6294 7081 6436 6899 W
7571 4676 7756 4563 S
1543 1348 1627 1150 W
5284 7705 5453 7515 N
5942 4576 6044 4427 W
1732 7915 1800 7784 W
3913 5534 4078 5362 N
6390 598 6522 486 W
725 5822 778 5625 W
7097 1022 7227 932 W
1392 683 1443 608 N
1699 425 1809 328 E
7585 1850 7703 1788 W
1990 3405 2058 3261 N
3099 6979 3184 6808 S
7316 1581 7469 1493 S
7081 4737 7255 4621 S
6200 4419 6282 4345 E
3099 4621 3216 4510 W
740 2732 815 2598 S
4131 4337 4307 4211 S
205 4934 355 4849 W
2291 5662 2460 5534 E
753 5083 890 4908 W
1195 3450 1278 3357 S
2466 7188 2639 7037 W
4581 7093 4650 6945 N
7550 2440 7713 2386 N
1560 5922 1707 5775 S
5583 2039 5705 1960 N
2196 3924 2384 3781 S
22 3951 153 3788 E
1795 4561 1990 4382 N
3565 3107 3678 2914 E
5323 316 5466 132 S
4296 3152 4470 2966 W
7069 5744 7134 5657 N
3737 7478 3848 7284 N
7637 2148 7815 1952 W
4900 7991 5032 7853 S
318 7607 517 7431 N
1547 5386 1621 5329 S
4393 7971 4472 7828 E
3627 4581 3824 4485 W
122 980 237 876 N
1046 1253 1186 1201 N
2010 6930 2117 6818 W
5692 7422 5799 7331 E
2347 7243 2404 7117 E
4633 2144 4693 1996 N
2746 6685 2836 6570 S
6852 5611 6993 5432 S
1917 4994 2106 4926 N
893 7710 952 7539 E
6304 735 6450 645 S
6197 6418 6331 6276 N
346 2811 426 2756 E
5716 2164 5915 2021 W
1887 6174 2067 6119 N
3364 7971 3497 7852 E
5671 7608 5723 7452 E
3314 6915 3498 6759 W
2072 7524 2258 7429 N
2848 2226 2936 2082 E
2825 1392 2989 1240 W
3996 1242 4079 1070 S
4955 174 5053 29 S
4722 2024 4836 1885 W
4886 4083 4953 3907 N
2240 6388 2350 6235 S
275 3859 372 3695 W
1208 1803 1322 1683 N
6379 6543 6517 6408 N
7222 2925 7316 2830 N
5627 1805 5811 1731 E
7589 3851 7760 3752 E
3182 7159 3271 7091 N
6852 5369 7029 5265 S
6759 6292 6900 6239 N
5843 3125 5957 3063 N
2778 4385 2849 4221 W
1045 701 1213 643 S
3824 5066 3917 4940 S
457 1085 513 996 N
1992 4003 2165 3946 S
3105 964 3274 863 W
2415 5361 2539 5239 N
2394 3585 2565 3491 E
7861 2082 7970 1913 N
2160 459 2308 305 S
4489 4971 4542 4826 W
1745 3245 1818 3075 E
1716 1805 1782 1740 W
1367 6123 1450 6069 E
6063 6650 6163 6532 W
2566 2194 2721 2007 E
3445 4251 3643 4058 N
4745 5766 4860 5576 W
4930 948 5127 877 E
1212 5244 1373 5106 E
6812 6757 7000 6683 E
1424 3112 1562 2958 S
2247 4740 2434 4689 S
6551 6136 6692 5990 E
7361 5422 7527 5227 S
771 2113 867 2058 N
2399 4964 2561 4795 E
6174 411 6230 273 S
3726 1864 3786 1704 W
528 3778 686 3688 N
6930 3331 6990 3215 N
5843 3930 6004 3781 S
4557 1189 4728 1133 E
3166 5319 3347 5186 W
2293 5355 2396 5223 S
6602 3678 6661 3557 S
7000 5187 7152 5093 N
4695 3458 4745 3262 E
4978 3404 5130 3294 W
3369 4509 3483 4331 N
353 2647 405 2473 S
4520 2969 4605 2877 E
4876 798 4946 624 N
1598 3651 1676 3600 S
6589 3870 6671 3756 W
3839 7869 3907 7805 W
7194 3771 7328 3678 S
6113 1835 6281 1668 E
4321 5138 4378 5079 N
3979 2693 4097 2575 N
902 3728 1000 3617 S
4723 525 4787 359 E
6111 5047 6240 4945 E
6013 834 6212 690 W
2514 3801 2648 3688 S
741 4454 817 4274 N
3873 7065 4031 6936 W
2761 839 2815 715 S
5208 2186 5366 2089 N
4506 380 4681 219 W
1270 2168 1333 2091 W
5737 2996 5813 2828 S
3706 2183 3791 2004 E
5031 3873 5133 3772 E
3921 6829 4083 6681 E
1563 3065 1756 2979 E
2283 6062 2437 5959 S
650 1740 824 1598 W
4139 886 4328 796 S
1382 3927 1495 3729 E
7110 2680 7214 2605 S
6190 5419 6367 5247 S
2146 2985 2249 2921 N
2176 1622 2292 1478 W
4804 7115 4898 6967 E
203 2426 384 2264 N
7077 1539 7136 1403 W
1577 771 1705 640 S
7483 1674 7662 1535 S
6682 5181 6812 5080 N
66 6146 118 5971 E
2300 1010 2397 839 S
4795 1810 4972 1681 N
3613 5107 3758 4936 E
4442 4799 4544 4729 E
5693 6389 5796 6204 N
4695 4430 4798 4351 N
6921 7029 7041 6839 W
4965 5293 5076 5227 W
3384 2755 3459 2571 S
2785 4562 2849 4510 W
797 7304 884 7187 S
4526 7472 4680 7314 N
3445 5964 3527 5890 W
2654 2765 2815 2568 N
4241 5659 4339 5493 E
7610 1506 7802 1436 S
546 5661 634 5512 S
7246 1153 7310 1095 W
1933 5950 2060 5848 N
2307 1192 2446 1112 W
706 3279 795 3210 N
620 3840 751 3780 E
4997 830 5193 636 E
6614 1622 6731 1463 N
1779 1010 1906 950 W
6156 4865 6327 4698 N
1829 4683 1958 4582 S
5069 1179 5250 1113 N
5340 5395 5480 5233 E
63 4695 202 4549 W
68 3021 174 2962 S
1585 2257 1686 2068 N
526 7672 591 7525 S
3831 3999 3950 3860 E
6516 6394 6577 6289 N
5014 3649 5156 3561 E
932 4458 1024 4359 S
3079 288 3179 236 N
1783 6494 1911 6383 E
4951 7606 5063 7511 S
3271 6307 3401 6204 E
6262 4299 6454 4113 S
5141 5829 5312 5736 E
3376 5341 3548 5247 W
830 6132 962 5934 S
7515 6382 7648 6246 S
3210 6055 3365 5960 E
4025 6621 4185 6495 W
4461 7480 4511 7304 S
6237 1645 6290 1519 S
2003 1573 2170 1520 E
1406 220 1516 101 N
4444 3552 4501 3477 S
934 222 1065 59 N
6733 6515 6795 6465 S
2071 6646 2225 6524 W
6325 328 6492 177 N
4560 7921 4646 7820 N
5035 3159 5105 3037 W
6213 7179 6317 7110 N
5583 5359 5766 5163 E
2356 3658 2504 3607 W
5288 5950 5453 5845 W
1310 535 1409 470 N
5160 7129 5340 7006 E
685 5529 780 5466 N
5254 3511 5448 3365 N
5734 2367 5794 2194 S
642 6206 802 6085 W
4358 892 4427 814 W
1611 6246 1792 6194 E
3797 2226 3948 2117 N
3997 6252 4123 6178 S
755 7694 846 7580 N
2619 7626 2754 7466 E
5789 7169 5933 7037 E
6369 6716 6482 6623 S
1840 2159 1993 2056 S
2138 218 2293 44 S
5180 3282 5376 3157 S
6869 1350 6965 1171 N
2075 1759 2132 1689 N
2902 7366 2995 7207 E
5519 5055 5701 4907 N
6464 7840 6644 7779 E
1848 210 1986 64 N
2162 3780 2345 3671 N
4021 344 4116 261 W
328 3405 444 3215 S
6254 2192 6425 2130 W
4037 5355 4101 5196 W
5742 736 5883 547 E
744 6816 926 6738 S
2211 4627 2407 4547 E
5191 3120 5307 3066 E
6534 2941 6673 2790 S
6493 1114 6691 1033 S
4120 1830 4224 1645 N
4195 3624 4349 3428 N
5445 4355 5605 4197 S
1979 3156 2051 3020 S
300 6438 360 6342 W
107 2767 295 2583 N
5724 5124 5777 4937 N
6987 6547 7047 6373 S
4864 2205 5017 2064 E
869 1699 981 1568 E
2726 2195 2810 2131 S
5447 6392 5636 6232 W
44 5906 135 5817 E
2984 5603 3084 5516 N
6557 7054 6631 6955 E
7841 5812 7992 5699 N
3837 6661 3932 6562 N
1396 4928 1546 4868 E
5414 7890 5470 7731 N
1398 6948 1565 6803 S
7380 1829 7570 1696 E
4049 779 4227 616 W
7602 5111 7692 4916 E
7586 5908 7720 5780 N
629 7957 801 7881 W
6236 5587 6343 5485 E
176 5143 309 5079 S
5659 6878 5790 6773 E
5663 1057 5784 967 S
6677 2947 6835 2860 N
2876 4520 3075 4424 N
6802 4250 6898 4076 N
2813 7908 2912 7791 N
2014 7352 2105 7281 E
6920 2555 6970 2459 S
2687 213 2828 123 E
3548 1752 3671 1660 N
7633 7777 7777 7648 W
4946 7112 5064 6990 N
3767 750 3873 567 N
1270 5866 1454 5815 N
6151 7466 6205 7269 E
7202 4024 7284 3878 E
2056 7864 2160 7787 E
2566 5479 2653 5299 N
114 7942 242 7881 N
6337 1270 6399 1143 W
7232 5550 7343 5485 E
3827 2941 3968 2857 E
5883 6314 6006 6248 S
7741 7174 7823 7120 N
779 5274 970 5167 N
4075 7841 4127 7730 E
4563 3174 4754 3044 E
1550 2500 1707 2339 S
7319 7732 7503 7663 E
4501 5365 4571 5279 E
2945 3455 3050 3258 N
7150 7005 7324 6904 W
1350 934 1457 770 S
5840 2427 5950 2338 S
2216 6169 2348 6116 W
5830 1634 5894 1557 W
579 6890 639 6794 E
5254 7960 5335 7810 N
7675 928 7783 759 S
3133 1239 3195 1087 E
4201 4744 4379 4650 N
3148 4806 3217 4704 S
3611 4389 3712 4324 N
6952 1870 7093 1708 S
1324 5389 1413 5292 S
126 353 178 154 N
2852 4284 2958 4160 N
2914 5362 3061 5221 S
6189 3128 6247 3029 N
4943 5838 5008 5663 S
4504 1907 4575 1719 W
2228 2785 2423 2618 E
5613 7846 5741 7705 S
4737 1549 4853 1426 W
87 3213 199 3155 S
1488 6439 1630 6297 E
6820 5174 6896 5086 N
7825 723 7882 561 W
1247 4754 1387 4626 W
4823 4952 4898 4797 S
1812 1293 1916 1147 S
2443 1665 2639 1511 N
1300 3387 1403 3258 S
3302 7478 3475 7296 E
149 1205 239 1148 N
6218 7415 6397 7243 W
5416 2737 5537 2587 N
5759 2774 5842 2680 N
7577 6206 7723 6151 E
4568 3424 4668 3344 N
2615 4176 2804 4087 W
6098 2844 6148 2746 S
3876 4928 4061 4829 W
3789 1212 3888 1124 W
3157 4372 3241 4178 N
7264 5381 7334 5331 N
3510 7964 3652 7784 N
6894 264 6985 190 S
5376 3117 5462 2950 W
545 5091 673 4999 N
5278 4147 5414 4078 N
976 3072 1122 2935 E
5969 6995 6142 6928 S
7051 4050 7121 3993 N
2950 7547 3099 7494 W
6191 2383 6390 2270 S
683 3190 747 3059 W
2510 6337 2677 6280 N
7174 823 7257 687 W
2014 4160 2069 4039 S
6905 1973 7073 1882 N
461 4676 531 4538 N
5456 1282 5542 1090 S
1523 2939 1717 2761 E
5520 252 5699 152 W
4335 6266 4524 6205 W
2595 4894 2699 4726 N
7470 5954 7536 5857 N
6730 6404 6798 6299 N
3593 5484 3784 5415 N
1907 4276 2006 4168 S
3709 6517 3844 6332 W
1216 1256 1312 1168 N
2753 1834 2943 1726 N
4736 5377 4907 5211 E
6381 2965 6479 2803 E
6146 3739 6223 3553 N
2562 3178 2734 3067 N
5961 7494 6022 7313 W
5974 7138 6047 7049 N
1585 6108 1693 6004 N
57 1365 121 1221 W
3725 1024 3869 852 S
2469 1416 2592 1270 W
6542 7534 6710 7439 N
4577 4598 4723 4454 W
3418 3936 3552 3835 E
2611 3268 2669 3186 W
5819 4311 5889 4159 S
7756 1175 7866 1071 S
6361 3232 6425 3050 W
1676 7702 1860 7534 S
6234 2954 6375 2775 E
4900 1549 5040 1429 W
801 4700 967 4647 N
7598 7522 7655 7455 W
4335 4303 4454 4229 E
3230 313 3308 124 E
4630 6300 4747 6122 E
4571 4719 4648 4605 S
1610 6714 1744 6652 W
1177 953 1230 842 S
7590 1772 7757 1702 W
2399 4269 2505 4203 E
2912 749 3055 569 S
1577 2058 1757 1871 W
2017 1177 2157 1051 S
5015 4354 5211 4245 W
7299 969 7475 877 N
1889 1626 1951 1573 W
1286 7595 1441 7496 E
691 542 868 443 W
5443 2350 5622 2216 W
1543 215 1671 126 E
5917 922 6028 848 E
4870 1021 4937 949 N
6072 5906 6180 5851 N
2699 3617 2760 3435 S
3230 2408 3308 2301 N
2490 5821 2618 5687 S
5107 4938 5175 4827 W
4075 3564 4141 3443 S
7576 602 7765 480 E
7719 1005 7901 936 E
3208 3714 3393 3537 E
102 7130 244 6964 N
3501 6778 3626 6601 N
4184 2833 4248 2678 N
5169 5564 5354 5478 W
4794 569 4936 431 W
3727 3918 3823 3751 E
7306 2530 7435 2334 S
4631 2484 4712 2388 W
3581 7254 3720 7079 W
2343 7075 2432 7006 E
4302 1391 4352 1338 N
3456 3789 3537 3684 S
7711 4862 7840 4752 W
5678 4143 5743 3966 S
599 7272 768 7153 E
7001 2106 7075 1982 E
5200 2938 5330 2791 E
3750 7759 3823 7609 W
6620 4127 6787 3981 E
4362 5914 4441 5857 N
7087 7062 7141 6961 N
1164 639 1217 570 E
5657 5903 5720 5770 W
646 2510 696 2454 W
4897 7244 5054 7175 N
6089 959 6166 866 E
4399 6189 4548 6060 N
5986 4096 6083 3942 E
6918 3094 7030 2935 N
4955 1365 5141 1191 E
7611 4443 7809 4375 S
2343 3110 2457 3000 N
795 2029 977 1952 S
3981 4797 4106 4660 N
3048 5175 3241 4994 S
6675 126 6809 66 N
5339 540 5416 385 N
3036 3952 3123 3765 S
2171 7052 2339 6984 S
1219 3034 1359 2951 N
2940 2253 3050 2198 E
6092 3434 6268 3325 S
7767 3526 7832 3449 S
120 6640 236 6497 S
2994 2180 3160 2025 E
7168 1812 7367 1660 N
4209 4540 4302 4386 W
726 6438 826 6304 N
4317 7136 4385 7070 W
7744 7509 7914 7311 S
260 1465 432 1277 E
4211 6741 4404 6636 N
7104 3438 7169 3349 E
7201 5222 7371 5081 W
2083 3378 2228 3300 E
4806 6627 4882 6461 E
6636 6782 6786 6601 S
41 7751 209 7680 N
6735 3619 6891 3553 S
1994 818 2096 623 W
2076 1968 2135 1805 E
4267 3364 4385 3276 S
6416 1031 6614 862 N
2951 904 3081 757 S
2226 7821 2406 7623 N
7390 2743 7514 2547 W
5587 947 5743 869 E
5275 7415 5438 7216 W
4443 7672 4567 7585 S
7310 2244 7438 2156 E
6683 7086 6814 7013 N
6457 2795 6527 2668 E
5887 361 5953 186 S
2845 6897 2964 6836 N
1149 6980 1233 6825 S
6110 2650 6186 2459 W
7448 6773 7582 6682 N
4268 7639 4382 7495 W
5187 2534 5323 2365 W
2465 5204 2602 5120 W
4500 6604 4607 6501 W
2859 6518 2933 6338 S
4073 5792 4127 5608 E
3127 6447 3206 6388 W
4192 5179 4255 5105 E
1252 7699 1422 7617 S
5876 89 6000 6 S
4802 6386 4932 6246 N
5458 7386 5531 7219 S
1971 500 2071 442 E
1823 1798 2012 1738 W
4222 1158 4419 1081 W
6848 1510 6898 1451 S
3021 6496 3091 6318 E
825 2360 883 2310 N
3556 535 3637 470 E
5418 6231 5515 6088 W
2695 1058 2798 985 S
3050 7802 3175 7662 E
608 7790 746 7666 S
6108 4101 6238 3924 S
361 5439 463 5316 N
7649 3697 7836 3531 S
4636 4337 4821 4153 N
7787 5327 7887 5164 W
7065 6595 7222 6490 S
5373 1058 5551 948 S
1200 1128 1344 1061 W
2610 6896 2666 6736 S
1422 3424 1600 3337 N
2090 5788 2284 5625 W
5731 6733 5797 6655 W
3551 7681 3745 7546 E
6083 525 6227 458 N
358 7935 500 7831 E
68 1963 260 1827 N
3439 1644 3589 1572 W
7855 2993 7980 2913 N
975 6342 1039 6182 S
4796 3268 4856 3090 W
733 6733 786 6663 N
3757 4468 3827 4409 E
3709 1684 3777 1579 N
1438 4108 1511 3969 W
180 4286 366 4175 W
1182 4212 1349 4061 S
735 2987 891 2800 N
807 7119 969 6925 W
1890 7160 2078 7065 E
4632 6786 4727 6729 W
6478 1801 6651 1717 S
5504 1697 5586 1549 W
312 1021 394 914 N
5843 4771 5935 4665 E
2873 3069 2948 2970 E
5083 6931 5240 6866 S
3677 508 3857 447 W
5211 1751 5364 1659 E
6698 675 6862 578 S
2234 3245 2366 3167 E
2361 2316 2482 2117 E
4430 2232 4496 2045 N
6749 3172 6894 3042 E
5709 5650 5799 5503 N
6968 7236 7156 7131 W
554 2095 705 1936 W
6205 6004 6394 5882 E
3317 1511 3464 1376 E
3576 2538 3698 2469 E
1121 3330 1263 3271 S
2748 1429 2816 1322 W
3448 5229 3630 5111 E
3690 5604 3746 5540 E
3821 7997 3943 7933 W
3193 635 3314 450 W
4113 2154 4180 2076 W
6707 4769 6861 4646 W
4230 7815 4377 7687 E
3807 2669 3877 2592 S
6128 6826 6239 6716 S
6321 5071 6379 4945 N
1186 3857 1334 3678 N
5523 5805 5587 5635 W
7367 7206 7538 7146 N
4480 5424 4628 5372 E
4355 172 4553 88 S
2197 6063 2263 5971 E
2587 5070 2766 5002 E
3144 3389 3252 3248 W
2244 6864 2374 6693 N
182 744 355 566 S
2693 738 2755 543 S
6462 4151 6541 4036 E
4425 5815 4553 5680 W
868 932 1049 872 S
4645 7744 4700 7621 E
5770 4507 5888 4350 E
4196 114 4312 26 S
642 7117 700 6986 W
5811 6725 5885 6608 E
3220 2796 3376 2710 W
1072 1983 1137 1932 N
3373 5428 3561 5372 W
3078 5905 3187 5755 W
5000 3011 5172 2950 S
1086 1606 1155 1507 S
1570 4076 1720 4004 E
4121 1622 4314 1557 S
5021 4083 5142 3961 N
2506 6235 2567 6056 W
1766 7240 1828 7086 N
724 197 858 83 S
3941 4308 4027 4202 W
1136 7087 1308 7011 E
5101 6829 5211 6658 E
796 5802 944 5633 E
3308 4798 3382 4732 S
5282 5130 5440 4959 S
6249 513 6376 458 E
7110 2476 7205 2308 N
4608 6655 4697 6475 W
7166 4448 7282 4277 W
4262 2876 4407 2798 E
301 6031 408 5924 N
7261 2828 7311 2729 S
5721 6112 5849 5926 N
6864 3939 7013 3832 S
5985 2118 6115 2055 S
722 2312 822 2166 S
5746 7803 5877 7636 N
2287 3559 2362 3407 N
2917 2663 3024 2609 S
5605 3874 5782 3738 E
2516 2639 2574 2524 S
6286 2678 6398 2576 E
6650 829 6734 726 S
5232 6212 5319 6093 S
5619 3097 5685 2922 S
4255 455 4445 352 W
7855 5630 7966 5502 S
2117 727 2176 634 S
3089 6697 3246 6560 N
832 1146 976 1092 S
1680 1672 1846 1484 N
3655 3748 3844 3678 S
388 7112 533 7002 S
2583 4975 2736 4907 E
3817 6084 3873 5934 E
7683 2910 7734 2713 E
3176 1576 3287 1455 N
4417 6972 4535 6880 W
363 4479 525 4331 S
2627 5546 2820 5487 S
3452 6428 3591 6367 S
3762 4323 3902 4248 S
1477 5959 1536 5815 E
6924 2303 7113 2251 W
3313 3329 3511 3187 N
6245 7828 6356 7687 E
2057 964 2150 834 E
4704 2618 4805 2546 E
4260 7372 4450 7279 S
5722 4278 5805 4188 N
90 1739 214 1645 W
6952 7744 7122 7660 N
4441 727 4540 587 E
2463 616 2513 459 S
7319 821 7385 635 E
2481 171 2653 68 W
687 1075 762 978 E
907 5554 987 5378 S
6451 2108 6572 1913 W
2552 700 2609 535 E
2122 2195 2219 2016 N
3128 5517 3198 5463 W
392 4201 586 4058 E
1414 2395 1482 2234 N
2679 2525 2816 2355 N
5257 1877 5453 1779 W
6182 3552 6266 3438 N
7543 7023 7599 6846 S
3111 1453 3175 1383 N
5524 3553 5659 3355 N
3513 247 3640 85 N
525 628 611 446 E
479 2958 611 2761 N
7583 7096 7639 7043 S
7389 5101 7484 4994 S
5522 2207 5600 2145 N
4370 6599 4421 6448 S
2750 1960 2891 1882 E
2386 3369 2447 3221 S
2660 1215 2795 1103 E
2056 3190 2112 3020 S
3892 5216 4007 5133 N
5492 6683 5606 6604 W
2676 7241 2743 7099 E
122 6173 197 6066 W
1090 2460 1182 2355 W
2235 2033 2379 1980 N
963 2800 1076 2657 N
7861 6747 7991 6681 E
1648 7210 1753 7130 W
5767 3164 5818 3009 W
28 5712 168 5521 E
3486 1265 3663 1214 W
6885 6481 6940 6395 E
6687 6586 6808 6526 N
2553 2974 2621 2801 E
6604 2485 6697 2412 N
5082 7380 5258 7277 N
6346 3567 6477 3375 N
6638 6966 6699 6821 N
6303 7503 6398 7433 S
5055 5665 5155 5490 N
5912 7875 6105 7699 S
5067 4604 5123 4471 N
5437 7070 5495 6963 E
6617 4338 6742 4248 W
4508 3705 4635 3563 E
7685 4128 7824 3938 E
6154 2070 6334 1885 N
7151 6154 7243 5957 W
695 3427 762 3366 N
6723 6113 6828 5946 S
6226 4522 6367 4427 N
7339 6062 7520 5998 N
5877 6111 5999 5952 W
6362 4855 6488 4755 N
258 2924 321 2792 E
3531 3644 3707 3480 N
5068 3489 5238 3423 W
5670 4875 5741 4758 W
6801 6925 6860 6863 S
7025 1224 7082 1071 E
7504 7364 7621 7255 W
1660 4994 1819 4821 N
3186 3882 3315 3810 N
3682 4220 3788 4116 N
4935 7851 5094 7722 W
1390 5262 1478 5140 N
7260 7398 7344 7337 W
3922 1412 3977 1327 W
2988 4896 3178 4833 W
609 926 719 817 W
835 6630 963 6571 S
6793 2318 6888 2194 E
1145 5621 1331 5448 S
5341 1628 5464 1500 W
4919 4848 4971 4650 N
4524 1360 4640 1279 E
2828 897 2915 802 N
6626 389 6712 253 S
4543 1646 4635 1544 S
6882 7938 7022 7765 N
4491 5963 4676 5856 N
5993 1419 6078 1353 W
6839 5051 6948 4881 S
3823 174 3946 15 N
2053 2709 2169 2644 N
1947 3920 2133 3843 E
494 4004 681 3946 N
1747 2297 1812 2245 W
4019 3268 4165 3106 E
3115 6288 3260 6198 N
1230 2468 1365 2320 S
4074 1440 4256 1264 W
7513 5724 7628 5591 N
316 392 506 195 S
6296 909 6384 743 W
2759 6235 2939 6176 S
3552 6275 3704 6149 E
2701 6549 2800 6471 E
1483 5614 1610 5492 N
3000 7192 3127 7098 W
3401 7701 3472 7536 W
4825 3523 4955 3372 W
3629 360 3716 259 S
4688 5543 4822 5444 W
4277 5250 4341 5176 N
1837 2845 2025 2769 W
7602 270 7747 155 W
2811 5094 2950 4953 E
3690 6730 3740 6604 N
7251 4979 7427 4904 S
3326 1786 3513 1670 S
6986 6048 7058 5965 N
4240 2314 4338 2139 N
2893 3745 3073 3639 E
2786 5884 2961 5772 N
7753 2824 7820 2654 S
2426 6703 2534 6525 E
197 3677 364 3572 S
6907 3583 7094 3506 W
1680 3966 1842 3904 N
7642 7986 7703 7921 N
165 3439 266 3316 E
2608 5268 2728 5184 N
4979 461 5102 331 S
389 2332 527 2244 S
290 7782 379 7691 W
295 1654 413 1531 E
6550 5789 6600 5691 S
5789 5151 5899 5068 S
5510 524 5650 468 W
6411 4569 6499 4382 W
3332 3173 3394 3002 W
977 7746 1116 7562 E
2032 4425 2180 4232 W
460 6787 564 6646 W
275 6815 353 6750 E
4674 8000 4783 7948 N
1306 3251 1405 3142 E
3290 1117 3478 950 W
5923 2712 6084 2529 E
3844 4801 3922 4737 S
221 295 293 212 E
7727 7886 7805 7794 N
1783 5734 1961 5591 E
1775 1729 1928 1673 W
5935 5220 6100 5027 S
2066 3654 2158 3591 W
5772 1905 5885 1818 N
1028 4165 1084 4021 S
1618 6854 1722 6757 E
4224 4905 4387 4778 W
177 7645 245 7468 S
5221 644 5347 559 N
7618 2545 7768 2487 W
370 2057 445 2003 N
3877 5731 4038 5615 E
3886 6238 3990 6180 S
1924 2429 2035 2235 N
5144 7885 5195 7812 W
60 1601 246 1484 N
423 5860 618 5734 E
7447 2365 7579 2222 W
407 6323 487 6241 E
4453 1074 4613 894 W
7629 6029 7776 5915 E
1820 3373 1986 3258 W
7265 1068 7357 1007 E
5364 7213 5547 7111 N
5480 791 5615 738 E
5130 6427 5205 6298 W
6655 5772 6813 5692 W
4188 3155 4278 3039 S
1480 2449 1544 2399 S
6641 4504 6797 4406 E
137 5333 268 5185 N
271 7417 453 7274 N
7810 4611 7947 4413 S
3492 1400 3634 1269 N
4145 6123 4249 6054 W
4325 4469 4427 4347 S
1058 5863 1229 5700 N
1665 3512 1799 3378 S
5031 2741 5091 2579 W
7511 4887 7699 4776 S
5805 5376 5968 5243 E
4653 5193 4717 4999 E
1013 5362 1185 5276 W
3283 4636 3359 4483 E
6216 6692 6313 6629 S
3606 732 3743 552 N
3515 4524 3604 4413 S
7803 1403 7943 1351 S
1540 4438 1593 4278 S
7573 3223 7679 3068 E
7195 7898 7379 7763 W
1948 6421 2009 6232 E
6368 5856 6442 5710 W
628 5416 754 5301 S
942 6873 998 6749 E
7383 223 7518 81 E
5404 4588 5483 4418 S
4248 3265 4406 3179 N
4159 5877 4298 5695 S
1531 5040 1627 4951 W
6248 3856 6376 3663 S
2583 4607 2776 4497 W
2618 7899 2775 7801 S
4225 6360 4304 6261 W
4445 5621 4626 5480 E
6979 3162 7150 3102 N
1143 6569 1244 6434 N
3656 206 3726 73 E
6367 5602 6463 5438 E
336 5605 462 5480 E
4239 668 4401 579 S
5605 2847 5722 2700 S
6540 3346 6716 3154 E
3858 7293 4020 7193 S
1796 5083 1995 4997 W
7105 2927 7209 2845 E
1220 6382 1393 6309 N
2108 4851 2217 4755 S
3115 7493 3269 7307 W
3943 602 4106 457 E
4636 3987 4729 3835 W
1953 6046 2134 5960 N
1495 7989 1605 7852 S
2222 783 2349 626 W
4767 2850 4823 2729 S
5679 7300 5774 7109 N
6478 3138 6674 3030 N
4173 1998 4227 1909 S
3502 2796 3627 2727 E
447 5221 600 5109 E
3049 1901 3120 1796 S
3304 4963 3480 4891 N
5657 118 5759 19 W
1587 989 1756 842 S
6737 1137 6936 958 E
3874 2429 3952 2287 N
4778 6122 4904 5956 W
1256 2043 1421 1943 S
2360 215 2464 130 N
4418 7741 4545 7689 E
1135 7508 1208 7353 E
5688 3552 5847 3500 E
7097 2986 7188 2931 N
1243 584 1301 426 E
1216 1632 1412 1460 S
6759 3529 6894 3355 W
7012 2618 7083 2542 S
1026 870 1119 756 W
4122 7368 4257 7260 E
205 2197 264 2070 S
886 4629 994 4555 S
2 5802 56 5730 E
4135 5604 4234 5409 N
1289 4464 1472 4288 W
2372 1385 2434 1298 E
3724 3279 3823 3141 W
801 7850 999 7754 E
3995 3828 4071 3744 S
4503 4563 4575 4431 W
4749 6752 4933 6657 S
5830 3322 5990 3142 E
3847 4153 3901 4076 W
7800 915 7989 790 N
7807 6594 7967 6456 N
7388 1468 7498 1306 W
56 6873 215 6731 N
5360 1233 5440 1082 W
2372 2483 2564 2379 W
