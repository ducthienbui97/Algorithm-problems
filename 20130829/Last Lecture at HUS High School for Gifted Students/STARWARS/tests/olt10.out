1
13
15
18
27
30
36
37
39
42
52
55
57
60
68
71
74
76
78
94
96
99
100
104
109
115
122
123
126
134
141
171
173
181
191
195
206
210
215
224
225
229
244
252
263
268
273
291
296
297
302
307
310
312
314
316
318
320
324
327
332
333
334
335
336
337
339
342
344
356
357
362
364
373
377
387
395
396
405
406
428
430
433
446
449
450
453
454
455
457
471
472
474
475
489
491
492
497
498
504
516
517
523
525
526
531
533
538
543
544
547
550
560
580
581
589
591
597
598
612
616
617
619
620
627
637
639
644
645
648
651
664
665
672
677
680
683
689
693
694
696
702
712
717
718
720
721
722
726
751
752
755
756
768
774
788
793
798
800
806
808
822
829
846
849
852
853
854
863
865
871
875
888
893
895
902
905
909
920
922
928
932
939
940
941
943
965
966
977
980
995
998
